docs/sources/flow/reference/components/otelcol.auth.sigv4.md

```diff
@@ -0,0 +1,171 @@
+---
+title: otelcol.auth.sigv4
+---
+
+# otelcol.auth.sigv4
+
+`otelcol.auth.sigv4` exposes a `handler` that can be used by other `otelcol`
+components to authenticate requests to AWS services using the AWS Signature Version 4 (SigV4) protocol. 
+See [here](https://docs.aws.amazon.com/general/latest/gr/signing-aws-api-requests.html) for more
+information about SigV4.
+
+> **NOTE**: `otelcol.auth.sigv4` is a wrapper over the upstream OpenTelemetry
+> Collector `sigv4auth` extension. Bug reports or feature requests will be
+> redirected to the upstream repository, if necessary.
+
+Multiple `otelcol.auth.sigv4` components can be specified by giving them
+different labels.
+
+> **NOTE**: The Agent must have valid AWS credentials as used by the 
+[AWS SDK for Go](https://aws.github.io/aws-sdk-go-v2/docs/configuring-sdk/#specifying-credentials).
+
+## Usage
+
+If the exporter URL does not contain enough information about the region and service, 
+the minimum arguments are:
+```river
+otelcol.auth.sigv4 "LABEL" {
+  region  = "REGION"
+  service = "SERVICE"
+}
+```
+
+In some cases the exporter URL does contain enough information about the region and service as 
+described [here](#arguments). In that case there is no need to specify any arguments to this component:
+```river
+otelcol.auth.sigv4 "LABEL" {
+}
+```
+
+## Arguments
+
+Name | Type | Description | Default | Required
+---- | ---- | ----------- | ------- | --------
+`region` | `string` | The AWS region for the service you are exporting to for AWS SigV4. | "" | no
+`service` | `string` | The AWS service for AWS SigV4. | "" | no
+
+Both `region` and `service` are required for the component to work, but if they are left empty their values 
+may be inferred from the exporter URL based on 
+[these](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/v0.63.0/extension/sigv4authextension/signingroundtripper.go#L121) rules:
+
+* If the exporter URL starts with `aps-workspaces` and `service` is empty, `service` will be set to `aps`.
+* If the exporter URL starts with `search-` and `service` is empty, `service` will be set to `es`.
+* If the exporter URL starts with either `aps-workspaces` or `search-` and `region` is empty, `region` .
+will be set to the value between the first and second `.` character in the exporter URL.
+
+For concrete examples, see the [examples](#examples) section.
+
+The AWS regions are listed [here](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Concepts.RegionsAndAvailabilityZones.html).
```

Comment:
Avoid calling links "here", instead try something like:

```suggestion
A list of valid AWS regions can be found on Amazon's documentation for [Regions, Availability Zones, and Local Zones](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Concepts.RegionsAndAvailabilityZones.html).
```

This will then align with the Google developer documentation recommendations for [Link text](https://developers.google.com/style/link-text).